tests/e2e/exchange/currencies/conftest.py

```
@@ -0,0 +1,57 @@
+
+
+@pytest.fixture
+def created_currency(currencies_service, currency_data, logo_fd):
+    currency = currencies_service.create(currency_data, file=logo_fd)
+
+    yield currency
+
+    try:
+        currencies_service.delete(currency.id)
+    except MPTAPIError as error:
+        print(f"TEARDOWN - Unable to delete currency {currency.id}: {error.title}")  # noqa: WPS421
```

Ruff (0.15.7)

[warning] Invalid rule code in `# noqa`: WPS421

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)